pyretis/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pyretis" %}
+{% set version = "0.9.2.dev2" %}
+{% set sha256 = "06c73d2778de7c65ae30464e2456637dfd326c9d94fd0bbfd557a789bff87816" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<3]
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  ## preserve_egg_dir: True
+
+requirements:
+  build:
```

Comment:
It looks like only `python` and `setuptools` are required to build the package (the `--single-version-externally-managed` flag, means that `install_requires` is ignored).